app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,10 @@
+from app.players.player import Player
+
+
+class Dwarf(Player):
```

Comment:
Dwarf also should inherits from ABC class

Reply:
![image](https://user-images.githubusercontent.com/103201010/218754358-201f1321-742f-47d5-a585-c337cf68612b.png)